example/nextjs/server.js

```diff
@@ -39,8 +61,34 @@ i18n
         // missing keys
         server.post('/locales/add/:lng/:ns', i18nextMiddleware.missingKeyHandler(i18n));
 
+        // Add language subpaths via i18n middleware
+        i18nextMiddleware.addRoute(
```

Comment:
is this needed? does not the * route below with the next.js handle work alone

Reply:
Yes, totally correct! Forgot about that block, I should have removed it.